### PR TITLE
Replaced fixed strings with translation for device-flash-dialogs

### PR DESCRIPTION
**Pull request by wimalopaan**
Replaced the fixed strings

### CODE REVIEW

File: radio/src/translations/cn.h.txt [hunk 1]
```diff
@@ -798,6 +798,7 @@
 #define TR_PATH_TOO_LONG               "路径太长"
 #define TR_VIEW_TEXT                   "查看文本"
 #define TR_FLASH_BOOTLOADER            "更新引导程序"
+#define TR_FLASH_DEVICE                TR("Flash Gerät","Flash Gerät")
```
Comment (raphaelcoeffic) [inline radio/src/translations/cn.h.txt:801]: Chinese looks like German now :-D

**Comment by raphaelcoeffic** [comment]
@wimalopaan can you please replace the one string in German with English in anything but German translation please? That makes it easier for translators. Other than that, it looks good to me, thx!

**Comment by wimalopaan** [comment]
> @wimalopaan can you please replace the one string in German with English in anything but German translation please? That makes it easier for translators. Other than that, it looks good to me, thx!

Sure! This is the first time I'm fiddeling with translations, but I hopefully got it right. I will insert english texts.

BTW: what is the reason to introduce explicit global storage to each translated string? Normally compilers would squeeze all usages of the same string-literal.

Wouldn't it make sense to qualify them all as `constexpr`.  But this may not be neccessary, because they are already  in the `text` (flash) section:
```
nm -CS --size-sort firmware.elf | grep STR_ | head
080e70cb 00000003 T STR_CH
080e724f 00000003 T STR_CV
080e750e 00000003 T STR_FM
080e75e2 00000003 T STR_GV
080e7606 00000003 T STR_ID
080e7b38 00000003 T STR_MS
080e7ee3 00000003 T STR_NO
080e803e 00000003 T STR_PPM_TRAINER
080e835a 00000003 T STR_SENSOR_A1
```

**Comment by raphaelcoeffic** [comment]
Thx. For the globals, I think this is mostly historical. I would prefer to have a look up table in fact, so we could dynamically load / map the translations (potentially supporting multiple languages without recompiling).

**Comment by wimalopaan** [comment]
> I would prefer to have a look up table in fact, so we could dynamically load / map the translations (potentially supporting multiple languages without recompiling).

Maybe this would be too much for the small b&w radios.

**Comment by wimalopaan** [comment]
Just a side note: the `#ifdef`s in `translations.h` are mostly useless, because the compiler will remove unused symbols. And some of the `#ifdef`s are inconsistent (as well in this PR, because some strings-literals are used only for COLORLCD), so they are at best misleading ...

File: radio/src/translations/en.h.txt [hunk 2]
```diff
@@ -874,6 +875,12 @@
 #define TR_FLASH_FLIGHT_CONTROLLER_BY_INTERNAL_MODULE_OTA "Flash FC by int. OTA"
 #define TR_FLASH_BLUETOOTH_MODULE      TR("Flash BT module", "Flash Bluetooth module")
 #define TR_FLASH_POWER_MANAGEMENT_UNIT "Flash pwr mngt unit"
+#define TR_DEVICE_NO_RESPONSE          TR("Device not responding", "Device not responding")
+#define TR_DEVICE_FILE_ERROR           TR("Device file prob.", "Device file prob.")
+#define TR_DEVICE_DATA_REFUSED         TR("Device data refused", "Device data refused")
+#define TR_DEVICE_WRONG_REQUEST        TR("Device acces problem", "Device acces problem")
```
Comment (pfeerick) [inline radio/src/translations/en.h.txt:881]: `access` (missing s at the end) - is the same for all instances of TR_DEVICE_WRONG_REQUEST

**Comment by pfeerick** [comment]
@richardclli Is this going to cause any conflicts with your translation stuff if merged?

**Comment by wimalopaan** [comment]
Is there an option to force the build script to use clang target ARM? 
I'm typically using GCC which is sometimes accepting more of the "new" features, here of the "constexpr-all-things" initiative.
I would like to use clang locally too to test my PRs.

**Pull request merged**